website_menu_by_user_status/README.rst

```diff
@@ -0,0 +1,100 @@
+============================
+Website Menu By User Display
+============================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:a2ff455071078badb7f6294ce0f6ac5e994792077e5e29a49ff0fa8bec350232
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fwebsite-lightgray.png?logo=github
+    :target: https://github.com/OCA/website/tree/17.0/website_menu_by_user_status
+    :alt: OCA/website
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/website-17-0/website-17-0-website_menu_by_user_status
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/website&target_branch=17.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+The module manages display website menu entries, depending if the user
+is logged or not. The selection of the display status can be chosen
+logged and/or not.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Go to Website > Configuration > Settings > Pages.
+2. Select the page for which you would like to hide the menu.
+3. In the Related Menu Items table, check whether the menu item is
+   visible for logged/unlogged users.
+4. remove default filter to edit website menu line
+
+Known issues / Roadmap
+======================
+
+-  2 flags will just toggle specific groups automatically [based on
```

Comment:
does this still exists as it reference v11 ?

Reply:
you need to activate developer mode and then go to website -> configuration->menus. There you can adjust the visibility.
Will change the readme. Thanks for the hint

Reply:
removed the roadmap section